src/main/java/mz/org/fgh/disa/notifier/service/EmailService.java

```diff
@@ -37,12 +36,13 @@ public EmailService(JavaMailSender javaMailSender) {
 	
 	public void sendEmail(String[] to, String subject, 
 			  			  String text,
-			  			  byte[] attachment, String attachmentName, String module, String startDate, String endDate) throws MessagingException, IOException {
+			  			  byte[] attachment, String attachmentName, String module, String startDate, String endDate, String repoLink) throws MessagingException, IOException {
```

Comment:
Since `attachment` and `attachmentName` are not used, better to remove them.
```suggestion
			  			  String module, String startDate, String endDate, String repoLink) throws MessagingException, IOException {
```